docs/modules/ROOT/nav.adoc

```diff
@@ -59,19 +56,27 @@
 ** xref:cluster-performance:near-cache.adoc[]
 ** xref:cluster-performance:imap-bulk-read-operations.adoc[]
 ** xref:cluster-performance:data-affinity.adoc[]
-include::architecture:partial$nav.adoc[]
 * Member/Client Discovery
 ** xref:clusters:discovery-mechanisms.adoc[]
 ** xref:clusters:discovering-by-tcp.adoc[]
 ** xref:clusters:discovering-by-multicast.adoc[]
 ** xref:clusters:discovering-native-clients.adoc[]
 include::kubernetes:partial$nav.adoc[]
 include::data-structures:partial$nav.adoc[]
-* Distributed Compute
+// Distributed Computing
+* xref:computing:distributed-computing.adoc[]
+** xref:data-structures:entry-processor.adoc[]
```

Comment:
This is also displayed in AP Data Structures | Map so gets same dupe nav item treatment...